projects/netgrif-components/src/lib/navigation/navigation.theme.scss

```diff
@@ -1,9 +1,33 @@
 @use '@angular/material' as mat;
 
 @mixin navigation-theme($theme) {
+    $primary: map-get($theme, primary);
     $accent: map-get($theme, accent);
 
     .active-navigation-route {
         background-color: rgba(mat.get-color-from-palette($accent), 0.3);
     }
+
+    .active-rail {
+        background: white;
+        border-bottom: 1px solid #dddddd;
+        border-top: 1px solid #dddddd;
```

Comment:
is light gray ok for all themes?

Reply:
its the border, its not taht much related to theme